src/storage/StorageFlags.cpp

```diff
@@ -18,5 +18,7 @@ DEFINE_int32(waiting_new_leader_interval_in_secs, 5,
              "interval between two requests for catching up state");
 DEFINE_int32(rebuild_index_batch_size, 1024,
              "The batch size when rebuild index");
+DEFINE_bool(offline, false,
+            "Ensure there is no writes currently.It's usually used for initialization");
```

Comment:
I think don't  need any config parameter for rebuild the index logic.

Reply:
after talk with dangleptr offline, we can use it to speed up build index with lots of data.

Reply:
> after talk with dangleptr offline, we can use it to speed up build index with lots of data.

OK, Let's ours discuss offline. I have some ideas about it.

Reply:
I means the offline is typed by users in console.  For example:
"REBUILD TAG INDEX name_index OFFLINE"